### DISCUSSION TITLE
[Feature] Phishing FP

### DISCUSSION THREAD

**Discussion by flaviozluca** [rspamd/rspamd#5062] [Q&A]
Friends,

Can you kindly check if these FP are normal? Very legit emails, FP due to the links of these popular voting systems.

Thanks.
[phishfp.zip](https://github.com/user-attachments/files/16273177/phishfp.zip)

**Comment by fatalbanana** [comment]
They are normal, it works [as it's supposed to](https://rspamd.com/doc/modules/phishing.html#principles-of-phishing-detection).

**Comment by flaviozluca** [comment]
Thanks, Andrew. Can you detail a bit more what it does, why it was caught? I can try to tell the legit sender.

No way to avoid these FP or lower the score? They are very useful, I understand, right?

Tks.

**Comment by fatalbanana** [comment]
Refer to the documentation: https://rspamd.com/doc/modules/phishing.html - it explains what this check does and how to configure exceptions.

https://rspamd.com/doc/faq.html#how-to-change-score-for-some-symbol

**Reply by flaviozluca** [reply]
Thanks.

**Reply by flaviozluca** [reply]
I tried to tell them, but notice 2 major systems causing FP with high score.

mercadolivre.com (huge auction site, biggest in latin america and 1 of the largest in the world)
and https://solucx.com.br/
